CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Updated installation documentation (#471)
+- The tutorial on Booleans now comes in markdown and CodeTour (#517)
```

Comment:
I guess this is an addition, not a fix of something broken?

Reply:
Good question. The tutorial was available before, but only in CodeTour. The new version is available in two formats. Yeah, that sounds more like an addition.